components/scream/ekat/src/ekat/util/ekat_string_utils.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef EKAT_STRING_UTILS_HPP
+#define EKAT_STRING_UTILS_HPP
+
+#include <string>
+#include <vector>
+
```

Comment:
Could you provide a paragraph comment here describing the higher-level idea behind this header. What is provides and why it is needed?

Reply:
done